pages/support/_enterprise-support.html

```diff
@@ -4,11 +4,9 @@ <h2>Enterprise Support</h2>
     <p>
       Gruntwork also offers an enterprise support plan that includes:
       <ul>
-        <li>Live chat in phone/video calls</li>
-        <li>Design reviews</li>
-        <li>Code reviews</li>
-        <li>Prioritized bug fixes</li>
-        <li>Guaranteed same-day response times</li>
+        <li>SLA on response times</li>
+        <li>Private shared Slack channel</li>
+        <li>Phone/video calls if we can’t resolve a problem async</li>
       </ul>
       For more information, please <a href="/contact">contact us</a>.
```

Comment:
Should we refer them to our contract terms before suggesting that they contact us? Or do we specifically want to drive all potential enterprise conversations through the sales channel?